doc/source/dev/roadmap-detailed.rst

```diff
@@ -427,7 +427,13 @@ The following improvements will help SciPy better serve this role.
     - nested ANOVA
     - analysis of covariance (ANCOVA)
 
-- Add additional tools for meta-analysis; currently we have just `combine_pvalues`.
+  Also, provide an infrastructure for implementing hypothesis tests.
```

Comment:
Most hypothesis tests have the same structure. Most tests get the basics right, but handling all the options that SciPy users expect (vectorization, `nan_policy`, masked arrays, etc.) is tricky. Contributors shouldn't have to re-invent the wheel each time, so I'd like to distill what I've learned about writing hypothesis tests over the past few years into an infrastructure that will make it easier to write feature-complete and robust new tests.